test/metabase/api/search_test.clj

```diff
@@ -1696,13 +1712,13 @@
                      :model/Card {reg-card-id :id} {:name (named "regular card")}
                      ;; DQs aren't searchable without a DashboardCard (see later test)
                      :model/DashboardCard _ {:dashboard_id dash-id :card_id card-id}]
-        (testing "The card data includes the `dashboard_id`"
```

Comment:
The frontend didn't use this field, so it's been removed. It was redundant with the id inside the :dashboard map.